salt/modules/zypperpkg.py

```diff
@@ -2548,6 +2563,22 @@ def _get_visible_patterns(root=None):
     return patterns
 
 
+def _find_ptf_packages(pkgs, root=None):
+    """
+    Find ptf packages in "pkgs" and return them as list
+    """
+    ptfs = []
+    cmd = ["rpm"]
+    if root:
+        cmd.extend(["--root", root])
+    cmd.extend(["-q", "--provides"])
+    for pkg in pkgs:
+        output = __salt__["cmd.run"](cmd + [pkg])
```

Comment:
```
[root@7635debe6eea /]# rpm -q --provides grep lua     
bundled(gnulib)
config(grep) = 2.20-3.el7
grep = 2.20-3.el7
grep(x86-64) = 2.20-3.el7
liblua-5.1.so()(64bit)
lua = 5.1
lua = 5.1.4-15.el7
lua(abi) = 5.1
lua(x86-64) = 5.1.4-15.el7
```

It looks like you can pass multiple packages to that command. Is it possible to only run this command once with all of the pkgs passed in to decrease the amount of commands we need to run?

Reply:
The problem is, I need to know which output belong to which package in the input line.
I will play a bit around with query format and see if we can do something about it.

Reply:
I found a solution and implemented it